apps/e2e/integration/reconciliation_order.spec.ts

```diff
@@ -1,90 +1,243 @@
-import { test, expect } from "@playwright/test";
+import { expect } from "@playwright/test";
 
 import { baseURL } from "./constants";
-import { getDashboard } from "@/helpers";
+import { testOrders } from "@/helpers/fixtures";
+import { createSupplierOrder } from "@/helpers/cr-sqlite";
+import { getDbHandle } from "@/helpers";
 
```

Comment:
Add: 
- should be able to select multiple supplier orders to reconcile at once
- should be able to continue reconciliation